packages/react/src/experimental/Await.spec.tsx

```diff
@@ -96,3 +96,30 @@ describe('useAwait', () => {
     await waitFor(() => expect(screen.queryByText(TEXT)).toBeInTheDocument())
   })
 })
+
+describe('Await component', () => {
+  it('should render child component with data from useAwait hook', async () => {
+    render(
+      <Suspense fallback="Loading...">
+        <Await options={{ key, fn: () => Promise.resolve(TEXT) }}>{({ data }) => <>{data}</>}</Await>
+      </Suspense>
+    )
+
+    expect(await screen.findByText(TEXT)).toBeInTheDocument()
+  })
+})
+
+describe('class AwaitClient clearError', () => {
+  beforeEach(() => awaitClient.reset())
+
+  it('clears promise & error for all cache without key', async () => {
```

Comment:
```suggestion
  it('should have clearError method to clear promise & error for all cache without key', async () => {
```

Reply:
oh thx! Reviews related to text were reflected [bb1b64a](https://github.com/suspensive/react/pull/169/commits/bb1b64ae81a2d006274c8a4a05e2b48735b3d791)